quiche/src/recovery/cubic.rs

```diff
@@ -679,9 +689,6 @@ mod tests {
         // Exit from the recovery.
         now += rtt;
 
-        // To avoid rollback
-        r.lost_count += MIN_ROLLBACK_THRESHOLD;
```

Comment:
This PR is pretty obtuse to review, so I just wanted to check, is the removal here was covered somewhere else?

Reply:
AFAICT this was never actually necessary, because the rollback doesn't happen either way.